packages/components-native/package.json

```diff
@@ -22,7 +22,8 @@
   },
   "dependencies": {
     "@jobber/design": "^0.38.2",
-    "react-native-svg": "^12.4.4",
+    "react-native-gesture-handler": "^2.10.0",
+    "react-native-svg": "^13.9.0",
```

Comment:
I'm guessing this is trying to match the mobile repo?

Reply:
Exactly